run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.tagCH4

```diff
@@ -1182,14 +1189,17 @@ ${RUNDIR_GLOBAL_Cl}
 #==============================================================================
 # Country/region masks
 #==============================================================================
+
+# Use updated files (v2024-01); these are COARDS-compliant.
 (((Scarpelli_Mexico
-1001 MEXICO_MASK     $ROOT/MASKS/v2018-09/Mexico_Mask.001x001.nc           MASK     2000/1/1/0 C xy 1 1 -118/17/-95/33
-1010 MEX_MASK_MIRROR $ROOT/MASKS/v2018-09/Mexico_Mask_Mirror.001x001.nc    MASK     2000/1/1/0 C xy 1 1 -118/17/-95/33
+1001 MEXICO_MASK     $ROOT/MASKS/v2024-01/Mexico_Mask.001x001.nc           MASK     2000/1/1/0 C xy 1 1 -118/17/-95/33
+1010 MEX_MASK_MIRROR $ROOT/MASKS/v2024-01/Mexico_Mask_Mirror.001x001.nc    MASK     2000/1/1/0 C xy 1 1 -118/17/-95/33
 )))Scarpelli_Mexico
 
+# Use updated files (v2024-01); these are COARDS-compliant.
 (((Scarpelli_Canada
-1002 CANADA_MASK     $ROOT/MASKS/v2018-09/Canada_Mask.001x001.nc           MASK     2000/1/1/0 C xy 1 1 -141/40/-52/85
-1011 CAN_MASK_MIRROR $ROOT/MASKS/v2018-09/Canada_Mask_Mirror.001x001.nc    MASK     2000/1/1/0 C xy 1 1 -141/40/-52/85
+1002 CANADA_MASK     $ROOT/MASKS/v2024-01/Canada_Mask.001x001.nc           MASK     2000/1/1/0 C xy 1 1 -141/40/-52/85
+1011 CAN_MASK_MIRROR $ROOT/MASKS/v20/Canada_Mask_Mirror.001x001.nc    MASK     2000/1/1/0 C xy 1 1 -141/40/-52/85
```

Comment:
There is a typo here `v20` should be `v2024-01`

Reply:
This is now fixed in https://github.com/geoschem/geos-chem/pull/1992/commits/e927528f7fe1f7c7b40768fd3c05626c0b822749.